src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  let price = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    price -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
```

Comment:
You can remove the `else` keyword here as per checklist item 5, because when the `if` condition on line 15 is true, a return statement will be executed and the function will exit. This simplifies the control flow and makes the code cleaner.